### ISSUE TITLE
hello world tutorial

### ISSUE THREAD

**Issue by piotrek2302** [dotnet/core#8852]
Problem encountered on https://dotnet.microsoft.com/en-us/learn/dotnet/hello-world-tutorial/create
Operating System: windows 10

I have installed the .NET SDK 7,0.402 and tried the first command in the basic hello world tutorial (https://dotnet.microsoft.com/en-us/learn/dotnet/hello-world-tutorial/create) to create new console 
`dotnet new console -o MyApp -f net7.0`
However after this I get an error message
`Could not execute because the application was not found or a compatible .NET SDK is not installed.
Possible reasons for this include:
  * You intended to execute a .NET program:
      The application 'new' does not exist.
  * You intended to execute a .NET SDK command:
      It was not possible to find any installed .NET SDKs.
      Install a .NET SDK from:
        https://aka.ms/dotnet-download`
Does anybody know what is wrong here? Is it my installation or something? I reinstalled .NET and restarted my PC and still this happens.

**Comment by liveans** [comment]
Hello,

Can you post the `dotnet --info` output here?

**Comment by piotrek2302** [comment]
```
Host (useful for support):
  Version: 5.0.15
  Commit:  3065735be7

.NET SDKs installed:
  No SDKs were found.

.NET runtimes installed:
  Microsoft.NETCore.App 5.0.15 [C:\Program Files (x86)\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.WindowsDesktop.App 5.0.15 [C:\Program Files (x86)\dotnet\shared\Microsoft.WindowsDesktop.App]

To install additional .NET runtimes or SDKs:
  https://aka.ms/dotnet-download
```
seems like i dont have any SDKs installed

**Comment by liveans** [comment]
> ```
> Host (useful for support):
>   Version: 5.0.15
>   Commit:  3065735be7
> 
> .NET SDKs installed:
>   No SDKs were found.
> 
> .NET runtimes installed:
>   Microsoft.NETCore.App 5.0.15 [C:\Program Files (x86)\dotnet\shared\Microsoft.NETCore.App]
>   Microsoft.WindowsDesktop.App 5.0.15 [C:\Program Files (x86)\dotnet\shared\Microsoft.WindowsDesktop.App]
> 
> To install additional .NET runtimes or SDKs:
>   https://aka.ms/dotnet-download
> ```
> 
> seems like i dont have any SDKs installed

Yes, can you make sure that SDK is installed properly, then you should restart the shell/command prompt and try the command again?

**Comment by piotrek2302** [comment]
Ok I will try every method to install SDK because I cannot seem to get it installed through the .exe file or installing it from the command line. Do you or anybody else have a solution to this?

**Comment by piotrek2302** [comment]
https://stackoverflow.com/questions/67049414/windows-or-visual-studio-2022-cant-find-the-latest-installed-net-sdk-due-to-bi/67049415#67049415
I managed to solve this issue by changing the order of enviroment variables in my windows settings. Took some time to figure out but I did it in the end. You can find the solution in the stackoverflow link.
